projects/rna/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+/yarn.lock
```

Comment:
This will only ignore `projects/rna/node_modules` and `projects/rna/yarn.lock`, neither of which should exist in the structure we're using for the monorepo.